packages/core/src/parsePheroApp/parseFunctionDefinition.ts

```diff
@@ -36,10 +36,12 @@ function parseFunctionName(
     | ts.Identifier
     | ts.StringLiteral
     | ts.NumericLiteral
+    | ts.BigIntLiteral
     | ts.ComputedPropertyName
     | ts.PrivateIdentifier
     | ts.ObjectBindingPattern
-    | ts.ArrayBindingPattern,
+    | ts.ArrayBindingPattern
+    | ts.NoSubstitutionTemplateLiteral,
```

Comment:
Question: should these new types be checked as well? I tried to create a function with a `BigIntLiteral` as name, but I didn't manage to without using an `ObjectBindingPattern`, which we already test for. But maybe there is a way I don't know about.

Reply:
Good catch!